models/chain-information.ts

```diff
@@ -0,0 +1,21 @@
+export interface ChainInformation {
+  chainName: string
+  description: string
+  platformChainId: string
+  subnetId: string
+  vmId: string
+  vmName: string // EVM, BITCOIN, ETHEREUM
```

Comment:
Might just want to use string literal unions here instead of comments:
```
vmName: 'EVM' | 'BITCOIN' | 'ETHEREUM'
```

Reply:
Yep good call, agreed.  Let's use string template literals when possible